main/.vuepress/config.js

```diff
@@ -118,7 +118,7 @@ module.exports = {
           ]
         },
         {
-          title: 'SES',
+          title: 'Endo and Hardened JavaScript',
```

Comment:
I wrote a whole section on Hardened JavaScript, so this is kinda confusing.

As I say, I'd rather prune the SES material in favor of links to the SES / endo repo docs.

Reply:
I’m somewhat torn on this. I for one would rather only update the lockdown options document in one place, but I would also like it to be discoverable through docs site search. If we hyperlink, what do we do about text search? Punt to Google? Google hasn’t yet indexed the SES error docs, and would probably be more likely to do so if they were embedded in the company website.